scanner/src/main/kotlin/experimental/ScanController.kt

```diff
@@ -59,7 +59,8 @@ class ScanController(
     private val provenanceResolutionIssues = mutableMapOf<Identifier, MutableList<OrtIssue>>()
 
     /**
-     * TODO: Docs
+     * A map of package [Identifier]s to a list of [OrtIssue]s, which can be used for occurring errors apart from the
```

Comment:
By "apart" do you mean "independently"?

Reply:
Yes, exactly. Should I change it to make it clear?

Reply:
I'd personally appreciate that, yes, but also then I'm still having slight difficulties understanding the sentence (but maybe that's just me): So is the map there in order to "share" `OrtIssues` between scanners on a per-`Identifier` basis? If that's the case maybe use something like:

> A map of `Identifer`s associated with a list of `OrtIssue`s that occurred during a scan with any scanner.

Reply:
Do you mean without any scanners?

Reply:
No, did not mean "without". I wanted to emphasize "*any* scanner" to explain that a contained issue might stem from any active scanner, not just a single one. I.e. it's a shared list that any scanner can add to. Or am I again misunderstanding something?

Reply:
> I.e. it's a shared list that any scanner can add to.

A shared list of `OrtIssue`s. That's exactly what the list is for. However, "_any scanner_" confuses me a bit, because there are more sources of error in the [`scan`](https://github.com/oss-review-toolkit/ort/blob/main/scanner/src/main/kotlin/experimental/ExperimentalScanner.kt#L141-L165) function and not only from "_any scanner_".

Reply:
Ok, so how about this then?

> A map of [Identifer]s associated with a list of [OrtIssue]s that occurred during a scan besides the issues created by the scanners themselves as part of the [ScanSummary].

Does that nail it?